modules/java/native/java-grammar.ym

```diff
@@ -86,6 +87,14 @@ java_dest_custom_options
   |
   ;
 
+java_dest_empty_options
+  : java_dest_empty_option java_dest_empty_options
+  |
+  ;
+
+java_dest_empty_option
+  : LL_STRING { ;  }
+
```

Comment:
This will not work when the multiple name/value pairs are passed to the options() argument. I don't know whether that syntax was deliberate though:

```
    java(options("foo", "bar", "baz", "bax"))
```
This passes foo=bar, baz=bax as arguments, but as there are not separators between the tokens, an empty option would mean:

```
    java(options("foo", `notdefined`, "baz", "bax"))
```
foo=baz
bax=<ignored>

This is not very intuitive either. If the list of arguments was intended and is documented, we shouldn't change that IMHO. @lbudai @juhaszviktor  what's your point on that?

If we don't want to change the list-of-name-value pairs support, then I would try to improve the error handling a bit, and request the user to enclose the value part in quotes.

This should be possible with a similar set of rules in the grammar, but instead of ignoring the value I would bail out with an error message.